quickstarts/runtime-options/functions/index.js

```diff
@@ -0,0 +1,44 @@
+const functions = require("firebase-functions");
+
+// [START runtimeMinInstances]
+exports.getAutocompleteResponse = functions
+    .runWith(
+        // Keep 50 instances warm for this latency-critical function
+        {
+          minInstances: 50,
```

Comment:
Nit: should we ever show a number this high? A copy-paste here would be very expensive.